test/library/packages/Itertools/Itertools.chpl

```diff
@@ -170,8 +170,8 @@ module Itertools {
       else
         coforall tid in 0..#numTasks {
           const workingIters = chunk(0..#times, numTasks, tid);
-          workingIters.translate(-workingIters.low);
-          yield(0..#(workingIters.high * arg.size),);
+          yield((workingIters.low * arg.size)..(((workingIters.high + 1) * 
+                arg.size) -1),);
```

Comment:
This looks like it'll fix both of the problems in the issue, thanks!

And thanks for wrapping the line to keep it under 80 columns.

However, there are better places to wrap the line.  It's split now in the middle of the computation of the high bound of the range, whiich is kind of mid-thought.  Can you find a better place?  (As a rule of thumb, the fewer parenthesized expressions that are split between lines, the better.)

Reply:
@cassella, 
```suggestion
          yield(workingIters.low * arg.size..#workingIters.size * arg.size,);
```
I think this seems good and increases readability of the code rather than breaking the line. Please suggest if I should use this?
Thanks,
R. Chinmay

Reply:
Yeah, that's even better than my idea!

Another option would have been introducing variables for parts of the computation so that the range expression would be shorter, but making the calculation simpler also means you don't have to do that.

Reply:
@cassella Sorry for the delayed response. I have made the changes suggested here. Please review.
Thanks,
R. Chinmay